docs/aws.md

```diff
@@ -478,7 +478,7 @@ The above snippet defines two volume mounts for the jobs executed in your pipeli
 
 <h4>Job queue not found</h4>
 
-**`JobQueue <YOUR_QUEUE> not found`**
+**`JobQueue <QUEUE> not found`**
```

Comment:
Nit, as a user I find much more intuitive `YOUR something` to understand it's a placeholder that I need to replace

Reply:
We're slowly moving towards a standardized placeholder across all doc sets. This is capitalized placeholders with underscores and no possessive adjectives. It's meant to make placeholders clearer and easier to scan. When a replacement action is required, we will start adding "Replace <PLACEHOLDER> with your XXXX" or something similar below the example. I didn't add a replacement instruction here as it's an example of what the error message looks like.